### PR TITLE
new example queries

### PR DESCRIPTION
**Pull request by danabauer**
See it here: https://dfhx9f55j8eg5.cloudfront.net/how-to/pr/11

- updated and new duckdb queries for examples page: https://labs.overturemaps.org/how-to/accessing-data/example-queries/
- add new Athena queries and other flavors of SQL - tk
- workflows - tk

### CODE REVIEW

**Review by jenningsanderson — APPROVED** [review]
❤️

File: docs/accessing-data/example-queries.mdx [hunk 1]
```diff
@@ -3,29 +3,48 @@ title: Example Queries
...
 import SeattlePlaces from '!!raw-loader!@site/src/queries/duckdb/seattle_places.sql';
 import AdminBoundaries from '!!raw-loader!@site/src/queries/duckdb/admin_boundaries.sql';
 
 ## DuckDB
 
-### 100 Buildings
+### Buildings in Detroit
 
-<QueryBuilder query={_100Buildings}></QueryBuilder>
+This query pulls building geometries and selected attributes in Detroit from the Overture buildings dataset in Azure blog storage.
```
Comment (jenningsanderson) [inline docs/accessing-data/example-queries.mdx:15]: blog -> blob

File: src/queries/duckdb/admin_boundaries.sql [hunk 2]
```diff
@@ -1,19 +1,20 @@
 LOAD httpfs;
 LOAD spatial;
+SET s3_region='us-west-2';
 
 CREATE VIEW admins_view AS (
     SELECT
         *
     FROM
-        read_parquet('s3://overturemaps-us-west-2/release/__OVERTURE_RELEASE/theme=admins/type=*/*', filename=true, hive_partitioning=1)
+        read_parquet('s3://overturemaps-us-west-2/release/2024-02-15-alpha.0/theme=admins/type=*/*', filename=true, hive_partitioning=1)
```
Comment (jenningsanderson) [inline src/queries/duckdb/admin_boundaries.sql:9]: _Nice catch_ on this one — we'll need to remember that some of these queries are _very_ version specific and be sure not to generalize for them.

Comment (danabauer) [reply src/queries/duckdb/admin_boundaries.sql:9]: Agreed for now. But I would like to move toward generalize as many example queries as possible over the next few releases.

**Pull request merged**